variables.tf

```diff
@@ -271,6 +284,24 @@ EOF
 
 }
 
+variable "http_proxy" {
+  type        = string
+  default     = ""
+  description = <<EOF
+HTTP proxy for downloading agents and vault executable on startup. Only necessary if allow_public_egress is false.
+This is only used on the initial startup of the Vault cluster, not in regular usage. If you configure Vault to
+manage credentials for other services, default HTTP routes will be taken.
+EOF
+}
+
+variable "create_external_load_balancer" {
```

Comment:
Reading through the code... I think it would be better if we made this "load_balance_type" with a default value of "external".

Reply:
Great idea, completely agree

Reply:
Called it load_balancing_scheme to match the variable in the forwarding rule, this way if it's something other than INTERNAL or EXTERNAL the user will get an error.